### PR TITLE
Feature/implement crystal platform

### PR DESCRIPTION
**Pull request by Erick9Thor**
Crystal platform script:

For now without animations only the logic. Tot test go to the TestScene and shoot the crytal. So if you try to dash before shoot you will fall, even the platform is visible. When you shoot the crystal, script actives the platform, this stay like 4s before starting shake.

### CODE REVIEW

**Review by monikamnc — APPROVED** [review]
Tested and working.
I don't know if maybe we can clean all the hash changes from the PR?

File: Gameplay/entities/crystals/CrystalExplosion.cpp [hunk 1]
```diff
@@ -55,11 +55,11 @@ void Hachiko::Scripting::CrystalExplosion::OnStart()
 
 void Hachiko::Scripting::CrystalExplosion::OnUpdate()
 {
-	ResourceAnimation* resource_animation = _explosion_crystal->GetComponent<ComponentAnimation>()->GetCurrentAnimation();
+	ComponentAnimation* resource_animation = _explosion_crystal->GetComponent<ComponentAnimation>();
```
Comment (alvarosoppelsa) [inline Gameplay/entities/crystals/CrystalExplosion.cpp:58]:
```suggestion
	ComponentAnimation* component_animation = _explosion_crystal->GetComponent<ComponentAnimation>();
```

**Review by alvarosoppelsa — APPROVED** [review]

File: Gameplay/entities/crystals/CrystalExplosion.cpp (hunk 1 above)
Comment (alvarosoppelsa) [reply Gameplay/entities/crystals/CrystalExplosion.cpp:58]: Added just a small change, but you have the approval already. Thanks!

**Comment by miquelmiro3** [comment]
Is the script Shake being used?

File: Gameplay/entities/crystals/CrystalExplosion.cpp (hunk 1 above)
Comment (Erick9Thor) [reply Gameplay/entities/crystals/CrystalExplosion.cpp:58]: Perfect lo fixe en otra pr en dev esta correcto mersi

**Pull request merged**